CONTRIBUTING.md

```diff
@@ -1,7 +1,72 @@
-# Sign your work
+## Contribution Guidelines
 
-The sign-off is a simple line at the end of the explanation for the patch, which certifies that you wrote it or otherwise have the right to pass it on as an open-source patch.
-The rules are pretty simple: if you can certify the below (from [developercertificate.org](http://developercertificate.org/)):
+### Pull requests are always welcome
+
+We are always thrilled to receive pull requests, and do our best to
+process them as fast as possible. Not sure if that typo is worth a pull
+request? Do it! We will appreciate it.
```

Comment:
On Tue, May 03, 2016 at 11:59:17AM -0700, Chris Aniszczyk wrote:
> The goal is for all OCI projects to share the same files, we have created a repo here:
> https://github.com/opencontainers/project-template

Sounds like a reasonable starting point, but this CONTRIBUTING.md is
currently too specific to apply to this repo (e.g. references to a
test suite we don't have).  Are downstream modifications like removing
test references or the leader-issue requirement allowed?  Does the
upstream project-template follow runtime-spec's “one sentence per
line”?  Should I be discussing this in
opencontainers/project-template#1?